### ISSUE TITLE
Feature: Add context and context list variables

### ISSUE THREAD

**Issue by volderette** [snowplow/snowplow-gtm-custom-template#19]
I've used [this snowplow gtm template](https://github.com/mmathias01/snowplow-gtm-templates) before and found it really helpful to have a context variable template. It would be great to integrate something similar in the snowplow gtm templates as well.

Link to repo with example templates:
https://github.com/mmathias01/snowplow-gtm-templates

**Comment by sahava** [comment]
Good idea - so basically variable templates to replace the need to use a Custom JS variable for returning the context arrays/objects?

**Comment by volderette** [comment]
Yes, exactly. This is an example of the mentioned templates:

![image](https://user-images.githubusercontent.com/1484860/83275868-140eb880-a1d0-11ea-9b42-3afa6dc64e1c.png)
